src/callbacks/onResize/removeYAxisTicks.js

```diff
@@ -0,0 +1,16 @@
+import { quantile } from 'd3';
+
+export default function addYAxisTicks() {
```

Comment:
@samussiah started looking at the log quantile stuff and saw this - should be removeYAxisTicks()